configuring.md

```diff
@@ -131,3 +131,13 @@ android {
   }
 }
 ```
+
+# ConscryptMode 
+
+Robolectric can currently either use Conscrypt and BouncyCastle or just BouncyCastle as the security provider. 
+In order to migrate tests over time, there is a [ConscryptMode](https://github.com/robolectric/robolectric/blob/master/annotations/src/main/java/org/robolectric/annotation/ConscryptMode.java) annotation that controls whether Conscrypt is loaded as the default security provider with BouncyCastle as backup. 
+
+- If ConscryptMode is `ON`, it will install Conscrypt and BouncyCastle.
+- If ConscryptMode is `OFF`, it will only install BouncyCastle.
+
+This is closer to to the way that it works on [real android](https://cs.android.com/android/platform/superproject/+/master:libcore/ojluni/src/main/java/java/security/Security.java;l=124-138).
```

Comment:
Let's add some detail on how this works. Something like:

```
Robolectric will search for a requested security feature from Conscrypt first. If it does not support it, Robolectric will try BouncyCastle second.
``` 

This can go as the last sentence.